docs/modules/user/pages/quickstart.adoc

```diff
@@ -364,6 +364,255 @@ docker run \
 --
 ======
 
+--
+Proxmox LXC + Nvidia (manual)::
++
+--
+
+[WARNING,caption=WARNING]
+====
+At the moment it is only possible to run Wolf inside a privileged LXC.
+====
+First you need to make sure your Nvidia drivers (version >= 530.30.02) are properly installed and loaded on your host.
+
+`nvidia-smi`
+```
++-----------------------------------------------------------------------------------------+
+| NVIDIA-SMI 550.107.02             Driver Version: 550.107.02     CUDA Version: 12.4     |
+|-----------------------------------------+------------------------+----------------------+
+| GPU  Name                 Persistence-M | Bus-Id          Disp.A | Volatile Uncorr. ECC |
+| Fan  Temp   Perf          Pwr:Usage/Cap |           Memory-Usage | GPU-Util  Compute M. |
+|                                         |                        |               MIG M. |
+|=========================================+========================+======================|
+|   0  NVIDIA GeForce RTX 4060 Ti     On  |   00000000:01:00.0 Off |                  N/A |
+|  0%   39C    P8              9W /  165W |       2MiB /  16380MiB |      0%      Default |
+|                                         |                        |                  N/A |
++-----------------------------------------+------------------------+----------------------+
+                                                                                         
++-----------------------------------------------------------------------------------------+
+| Processes:                                                                              |
+|  GPU   GI   CI        PID   Type   Process name                              GPU Memory |
+|        ID   ID                                                               Usage      |
+|=========================================================================================|
+|   No running processes found                                                            |
++-----------------------------------------------------------------------------------------+
+```
+
+`ls -l /dev/nividia*`
```

Comment:
I guess it's a typo `/dev/nividia*` should be `/dev/nvidia*`

Reply:
thanks, will fix.